docker-library/apache-php-mysql/php.ini

```diff
@@ -4,4 +4,7 @@ error_log=/var/log/httpd/php-error.log
 display_errors=Off
 display_startup_errors=Off
 date.timezone=UTC
-
+; increase upload size for PHP
+post_max_size=20M
+upload_max_filesize=20M
```

Comment:
In briefly, post_max_size = upload_max_filesize * file counts. 
Upload_max_filesize set limit for the upload file but post_max_size is checked for all kind of POST data.
In according to customer requirement, the import database file should be one.